### PR TITLE
[compiler/luci-pass-value-test] Enable RemoveUnnecessaryStridedSlice_000

### PR DESCRIPTION
**Pull request by a-zh**
This commit enable RemoveUnnecessaryStridedSlice_000 test case

Draft: https://github.com/Samsung/ONE/pull/5876

Signed-off-by: Anton Zhukov <anton.v.zhukov@gmail.com>

### CODE REVIEW

File: compiler/luci-pass-value-test/test.lst [hunk 1]
```diff
@@ -22,3 +22,5 @@ addeval(Net_TConv_BN_001 fuse_batchnorm_with_tconv)
 addeval(Net_InstanceNorm_001 fuse_instnorm)
 addeval(Net_InstanceNorm_002 fuse_instnorm)
 addeval(Net_InstanceNorm_003 fuse_instnorm)
+addeval(Net_StridedSlice_StridedSlice_000 remove_unnecessary_strided_slice)
+
```
Comment (seanshpark) [inline compiler/luci-pass-value-test/test.lst:26]: This is unnecessary empty space

Comment (a-zh) [reply compiler/luci-pass-value-test/test.lst]: Done

**Comment by a-zh** [comment]
@struss @seanshpark **PTL**

**Comment by mhs4670go** [comment]
@a-zh It failed. Did you pass the test in your local environment?

**Comment by seanshpark** [comment]
> 04:31:18 make[2]: *** No rule to make target 'compiler/common-artifacts/Net_StridedSlice_StridedSlice_000.circle', needed by 'compiler/luci-pass-value-test/Net_StridedSlice_StridedSlice_000.pass.circle'.  Stop.

Please rebase to master

**Comment by a-zh** [comment]
> @a-zh It failed. Did you pass the test in your local environment?

Yes, sure. I checked it under this draft environment https://github.com/Samsung/ONE/pull/5876

**Comment by a-zh** [comment]
> > 04:31:18 make[2]: *** No rule to make target 'compiler/common-artifacts/Net_StridedSlice_StridedSlice_000.circle', needed by 'compiler/luci-pass-value-test/Net_StridedSlice_StridedSlice_000.pass.circle'.  Stop.
> 
> Please rebase to master

Rebased

**Comment by seanshpark** [comment]
> 18:36:22 [ 95%] Generate Net_StridedSlice_StridedSlice_000.pass.circle with --remove_unnecessary_strided_slice
18:36:22 Invalid argument. You've given more positional argument than necessary.
18:36:22 What circle2circle does: circle2circle provides circle model optimization and transformations

**Comment by seanshpark** [comment]
> 20:18:05 [ 95%] Generate Net_StridedSlice_StridedSlice_000.pass.circle with --remove_unnecessary_strided_slice
20:18:05 Invalid argument. You've given more positional argument than necessary.

**Comment by a-zh** [comment]
@struss @seanshpark Please, take a look.

**Review by struss — APPROVED** [review]
LGTM

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**